packages/aws-lambda/layer/bin/publish-layer.sh

```diff
@@ -160,16 +165,13 @@ if [[ -z $AWS_ACCESS_KEY_ID ]] || [[ -z $AWS_SECRET_ACCESS_KEY ]]; then
   echo Warning: AWS_ACCESS_KEY_ID or AWS_SECRET_ACCESS_KEY are not set. This might be okay if you have set up AWS authentication via other means. If not, the AWS cli commands to publish the layer will fail.
 fi
 
-echo "step 1/9: fetching AWS regions (skipping, using fixed list of regions for now)"
-
-# Actually, this should give us the regions where the Lambda service is provided:
-# REGIONS=$(aws ssm get-parameters-by-path --path /aws/service/global-infrastructure/services/lambda/regions --output text --query "Parameters[].Value" | tr '\t' '\n')
-# But for some reason, publishing to all of these regions does not work. In particular, the
-# following regions either require special authorization/subscription status or don't support Lambdas: ap-east-1 me-south-1 ap-northeast-3
+echo "step 1/9: AWS regions"
 
 if [[ -z $SKIP_AWS_PUBLISH_LAYER ]]; then
   echo Will publish to regions:
   echo "$REGIONS"
+else
+  echo Publishing to AWS will be skipped.
 fi
 
 echo "step 2/9: Prepare build enviornment"
```

Comment:
A small typo, `enviornment`, I know it's not part of this PR, but it's a minor change.